cms/wp/db/backup/download.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Download the latest database backup file.
+ *
+ * @package wordpress
+ */
+
+namespace Deployer;
+
+task('wp:db:backup:download', function () {
+  $latestBackup = run("find {{backups}}/ -type f -printf '%T@ %p\n' | sort -n | tail -1 | cut -f2- -d\" \"");
+  download($latestBackup, "{{backups}}/$stage.sql.gz" );
```

Comment:
I don't think `$stage` is defined

Reply:
Stage is defined in deployer and is the machine name (poorly named by deployer) of the system being deployed to.  But this specific code is a carry over from an older backup script that used stage differently and it should be changed here.

Reply:
Gotcha, as long as the variable itself is defined here before it's used.